Microsoft.Toolkit.Uwp.Input.GazeInteraction/GazeHidParsers.h

```diff
@@ -0,0 +1,71 @@
+#pragma once
```

Comment:
missing license header

Reply:
fixed, as well as other files in library that were missing headers